public/js/themes.js

```diff
@@ -1,47 +1,26 @@
-// Managing theme changes:
-const darkId = 'dark-theme-css'
-const lightId = 'light-theme-css'
-const darkEl = document.querySelector(`link[id="${darkId}"]`)
-const lightEl = document.querySelector(`link[id="${lightId}"]`)
-
-function setCssDark() {
-  darkEl.media = ''
-  lightEl.media = 'none'
-}
-
-function setCssLight() {
-  lightEl.media = ''
-  darkEl.media = 'none'
-}
-
-// Sets up the needed css
-// Will be calling the function once browser parsed the icon and link elements
-function setCssBeforeLoading() {
-  const styleSheet = localStorage.getItem('stylesheet-key')
-  if ((typeof styleSheet == 'undefined') || (styleSheet == null)) {
-    localStorage.setItem('stylesheet-key', lightId)
-    setCssLight()
-  }
-  else if (styleSheet === lightId) {
-    setCssLight()
-  }
-  else {
-    setCssDark()
+function getCookie(cname) {
+  const name = cname + '='
+  const decodedCookie = decodeURIComponent(document.cookie)
+  const ca = decodedCookie.split(';')
+  for (let i = 0; i < ca.length; i++) {
+    let c = ca[i]
+    while (c.charAt(0) == ' ') {
+      c = c.substring(1)
+    }
+    if (c.indexOf(name) == 0) {
+      return c.substring(name.length, c.length)
+    }
   }
+  return ''
 }
 
-// Calling the function before parsing ends and css is applied to avoid flickering
-setCssBeforeLoading()
-
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 function changeTheme() {
-  const styleSheet = localStorage.getItem('stylesheet-key')
-  if ((typeof styleSheet == 'undefined')  || (styleSheet == null) || (styleSheet === lightId)) {
-    localStorage.setItem('stylesheet-key', darkId)
-    setCssDark()
-  }
-  else {
-    localStorage.setItem('stylesheet-key', lightId)
-    setCssLight()
+  if (getCookie('theme') === 'dark') {
+    document.cookie = 'theme=light;path=/;SameSite=Lax'
+
+  } else {
+    document.cookie = 'theme=dark;path=/;SameSite=Lax'
   }
```

Comment:
I guess you can solve this in one line instead of the if
```
 document.cookie = `theme=${getCookie('theme')};path=/;SameSite=Lax`
 ```

Reply:
The purpose of this is to change from light to dark, so i don't think that approach will work. I refactored to something that uses a clearer style.

Reply:
You're right, sorry